### DISCUSSION TITLE
Hour and Minute to Date Auto Template Creator

### DISCUSSION THREAD

**Discussion by tidypy** [sql-bi/Bravo#474] [Ideas]
Is there a way to add "Hour and Minute"  to the AutoCalendar Creator?  
Or is this a mute point and I need to create the HOUR out of the column in the desired table?

Example:  GET Data for [Date.Modified.modifieddatetime from TableXYZ] is the System date modified field;
and contains format  YYYY:MM:DD**T**hh:mm:ss.sssss

![image](https://user-images.githubusercontent.com/50213641/187518126-0b09c440-6687-46e4-8e79-c718d4277d4b.png)

![image](https://user-images.githubusercontent.com/50213641/187519147-2466d1ff-5321-4f12-9a5b-657476a7a9eb.png)

![image](https://user-images.githubusercontent.com/50213641/187519966-dd1660f7-1586-4d98-8aa4-65c437669bc3.png)

**Comment by tidypy** [comment]
![image](https://user-images.githubusercontent.com/50213641/191286490-c9870bed-b39c-4beb-9316-edd0f7676f45.png)

and some prebuilt measures:  
![image](https://user-images.githubusercontent.com/50213641/191286550-3515423c-1aa7-4622-9147-0b8a8348575f.png)

**Reply by marcosqlbi** [reply]
Please check the reply to #489 discussion, it's the same topic (sorry for the delay in replying to this thread).
